### ISSUE TITLE
Duke Nukem 3D (1996)

### ISSUE THREAD

**Issue by TimberlAndRE-de** [exoscoriae/eXoDOS#4269]
wysiwyg found

- [x] official strategies & secrets guide (quality upgrade)

@exoscoriae 
there was a CD included with the book containing demos, maps, tutorial videos and stuff ... 
an ISO was delivered with this scan but not sure if and how we include such discs ? I left it in the folder to let you decide

- [x] unauthorized Prisma guide

added to the github folder

**Comment by exoscoriae** [comment]
added guide and DMO files to game, with menu to choose from them. However, they loose sync over time. Possibly an emulation issue? I tried various cycles and cores, no change. The DMO files only appears to work with v1.3D. V1.4 and 1.5 refused to play them. As there is no registered version before 1.3D, I have to assume we have the correct version of the game. I also tried a clean, fresh install of 1.3D to make sure any tweaks we may have made (such as the apogee sound patch) did not inadvertently mess them up.

Left them in, with a note. Hopefully one day a fix show up. Would be good to test this on original hardware, to see if the issue is with emulation or not.
